tests/spread/plugins/v2/colcon-ros2-hello/task.yaml

```diff
@@ -0,0 +1,51 @@
+summary: >-
+  Build, clean, build, modify and rebuild, and run hello
+  for ros2/colcon.
+
+kill-timeout: 30m
+
+environment:
+  SNAP/colcon_ros2_foxy_rlcpp_hello: colcon-ros2-foxy-rlcpp-hello
+  SNAPCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS: "1"
+
+systems:
+  - ubuntu-20.04-amd64
```

Comment:
you are missing the systems that would trigger this on google and the system we use for lxd

Reply:
fixed!

Reply:
the list is still missing "ubuntu-20.04" for lxd

Reply:
SMH, fixing

Reply:
fixed, for real this time :facepalm: